src/deploy-cromwell-on-azure/scripts/startup.sh

```diff
@@ -115,13 +115,13 @@ rm -f .env && for key in "${!kv[@]}"; do echo "$key=${kv[$key]}" >> .env; done
 
 if [ -n "$mysql_server_name" ]; then
     fully_qualified_server_name="$mysql_server_name.mysql.database.azure.com"
-    mysql_db_name=${kv["MySqlDatabaseName"]}
-    mysql_user_login=${kv["MySqlUserLogin"]}
-    mysql_user_password=${kv["MySqlUserPassword"]}
-    write_log "Checking if database $mysql_db_name is locked"
-    lockTableExists=$(mysql -u "$mysql_user_login" -p"$mysql_user_password" -h "$fully_qualified_server_name" -P 3306 -e "SELECT EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE table_schema = '$mysql_db_name' AND table_name = 'DATABASECHANGELOGLOCK') AS '' ")
-    [[ "$lockTableExists" -eq "1" ]] && isLocked=$(mysql -u "$mysql_user_login" -p"$mysql_user_password" -h "$fully_qualified_server_name" -P 3306 -e "SELECT CONV(LOCKED, 2, 2) AS '' FROM $mysql_db_name.DATABASECHANGELOGLOCK WHERE ID = 1") || isLocked=0
-    [[ "$isLocked" -eq "1" ]] && write_log "Removing lock from $mysql_db_name" && mysql -u "$mysql_user_login" -p"$mysql_user_password" -h "$fully_qualified_server_name" -P 3306 -e "UPDATE $mysql_db_name.DATABASECHANGELOGLOCK SET LOCKED = 0, LOCKGRANTED = null, LOCKEDBY = null WHERE ID = 1" || write_log "$mysql_db_name was not locked"
+    mysql_db_name=${kv["PostgreSqlDatabaseName"]}
+    mysql_user_login=${kv["PostgreSqlUserLogin"]}
+    mysql_user_password=${kv["PostgreSqlUserPassword"]}
+#    write_log "Checking if database $mysql_db_name is locked"
+#    lockTableExists=$(mysql -u "$mysql_user_login" -p"$mysql_user_password" -h "$fully_qualified_server_name" -P 3306 -e "SELECT EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE table_schema = '$mysql_db_name' AND table_name = 'DATABASECHANGELOGLOCK') AS '' ")
```

Comment:
Is relieving the lock still necessary for postgres? If so- 
Executing postgres commands from the command line is a little different compared to mysql. As you can see from CreateMySqlDatabaseUser() in Deployer.cs, I was able to get something workable using `sudo -u postgres psql postgresql://` but I couldn't get the command to work without specifying a database, and this code executes at the user level not db. postgres doesn't have a -p for password option, only an interactive prompt with -W, which won't work for this case.

Reply:
@tonybendis to take a look after today's meeting